internal/rest/k8s_utils.go

```diff
@@ -104,9 +95,28 @@ func (rest *RestOperations) SyncObjectStatuses() {
 							Key:                lib.SyncStatusKey,
 							VirtualServiceUUID: vsCacheObj.Uuid,
 						})
+				} else if len(poolCacheObj.ServiceMetadataObj.NamespaceServiceName) > 0 {
```

Comment:
We are updating allServiceLBUpdateOptions again in line no 110, would that add a duplicate ?

Reply:
that is corresponding to vs cache svc metadata. This one is when corresponding to pool metadata. Had made an enhancement in pool svc metadata sometime back for `servicesAPI: true` so this takes care of that. The other one is for dedicated L4 VSes.